bin/lib/preview.js

```diff
@@ -46,17 +46,17 @@ function formatOptions (options) {
 }
 
 function formatBody (body) {
-  var arr = []
+  const arr = []
   arr.push({ column1: justify('body'), column2: '' })
   arr.push({ column1: '', column2: '' })
-  body.split('\n').map(function (line, i) {
+  body.split('\n').forEach(function (line) {
```

Comment:
i pushed one more commit up to get tests passing again.

this is the only line change that wasn't a straightforward `s/var/const || let`. it resolves the lint error below.

```bash
/Users/john/code/gh-release/bin/lib/preview.js
  52:24  error  Array.prototype.map() expects a return value from function
```